src/renderer/shared/lib/utils/functions.ts

```diff
@@ -20,6 +20,23 @@ export function nullable(value: unknown): value is null | undefined {
   return value === null || value === undefined;
 }
 
+type NonNullableMap<T extends Record<string, unknown>> = {
```

Comment:
Can use `PropertyKey` instead of string - `type PropertyKey = string | number | symbol`
Same for `nonNullableMap`